roboteam_ai/src/control/shotControllers/ShotController.cpp

```diff
@@ -61,10 +71,10 @@ void ShotController::determineGenevaAndPosition(const world::Robot &robot, const
         if (genevaDifference != 0) {
             genevaIsTurning = true;
             // each turn should increase the time which the geneva is turning
-            secondsToTurnGeneva = genevaDifference * 0.5;
+            secondsToTurnGeneva = genevaDifference * 0.3;
```

Comment:
this might depend on real life / grSim ( and == magic number )

Reply:
yes. Have to calculate for that.

Reply:
will make an issue (it needs to become a function, since switching multiple states is faster per state than only one state, since the geneva has a pid)